AllASPFuncs.py

```diff
@@ -1449,11 +1450,21 @@ def calc_mags_and_err(flux, sigma_flux, band, zp = None):
             'Y106': 302.275,
             'Z087': 101.7}
 
+    flux = np.atleast_1d(flux)
+    sigma_flux = np.atleast_1d(sigma_flux)
+
     area_eff = roman.collecting_area
     zp = roman.getBandpasses()[band].zeropoint if zp is None else zp
-    mags = -2.5*np.log10(flux) + 2.5*np.log10(exptime[band]*area_eff) + zp
-    magerr = 2.5 * sigma_flux / (flux * np.log(10))
-    return mags, magerr
+    positive = (flux > 0)
+    mags = np.zeros_like(flux)
```

Comment:
Use singular `mag`.  As a general rule, don't use plurals with variable names because it's hard to remember which is singular and which is plural, and then you have `magerr` but not `magserr`.

Reply:
I suppose this means I would want the function name to be `calc_mag_and_err`.